xdsl/interactive/app.py

```diff
@@ -274,6 +283,26 @@ def watch_available_pass_list(
                     ),
                 )
 
+    def update_selected_passes_list_view(
+        self, pass_pipeline: tuple[tuple[type[ModulePass], PipelinePassSpec], ...]
```

Comment:
if this is always called with `self.pass_pipeline`, it feels like we can drop the function argument